leaflet/index.d.ts

```diff
@@ -1292,8 +1346,10 @@ declare namespace L {
         addHandler(name: string, HandlerClass: () => Handler): this; // HandlerClass is actually a constructor function, is this the right way?
         remove(): this;
         createPane(name: string, container?: HTMLElement): HTMLElement;
-        getPane(pane: string): HTMLElement;
-        getPane(pane: HTMLElement): HTMLElement;
+        /**
+         * Name of the pane or the pane as HTML-Element
+         */
+        getPane(pane: string | HTMLElement): HTMLElement;
```

Comment:
I have not give it the `Content` type, because here it is the name of the pane or the HTML-Element of the pane root element. The string data would not be the content, but from the data-types point of view it is the same, that's right.

Would you still prefer the `Content` type? Then I will change it later...